gather_istio.sh

```diff
@@ -232,7 +225,23 @@ function main() {
       inspectNamespace "${dpn}"
       getEnvoyConfigForPodsInNamespace "${cpNamespace}" "${ir}" "${dpn}"
     done
-done
+  done
+
+  # Inspect all Kialis
+  for kialiNS in $(oc get Kiali -A -o jsonpath="{.items[*].metadata.namespace}"); do
+    echo
+    echo "Inspecting Kialis in ${kialiNS} namespace"
+    inspect "Kiali" "${kialiNS}"
+    inspectNamespace "$kialiNS"
```

Comment:
Can there be two Kialis in the same namespace?

Reply:
I'm not sure, I asked Kiali team. But even if it is possible, `oc adm inspect kiali -n <namespace>` should get all of them in the namespace.

Reply:
It's possible:
yes, you need to have different spec.deployment.instance_name  values. This allows you to install Kiali Server in teh same namespace multiple times. If you don't want to put multiple Kiali Servers in the same namespace, but just want to put all your Kiali CRs in the same namespace, then I think that is also doable - you just have to make sure you tell it to put Kiali in a different place (spec.deployment.namespace)... not sure I never tried that.. but I do know different instance_name values will install multiple Kialis in the same namespace (I have a [molecule test](https://github.com/kiali/kiali-operator/tree/master/molecule/instance-name-test) that tests that)